index.js

```diff
@@ -81,12 +81,12 @@ exports.getClickableButtons = () => {
   return lib.getClickableButtons();
 };
 
-exports.getEditorState = (fallback) => {
```

Comment:
Will this necessitate any changes to the client?

Reply:
should be okay, since I think the client is already not passing this argument.